cmd/kubernetes/kubernetes.go

```diff
@@ -79,13 +79,18 @@ func runApiServer(etcdClient tools.EtcdClient, addr net.IP, port int, masterServ
 	handler := delegateHandler{}
 
 	etcdStorage, err := master.NewEtcdStorage(etcdClient, "", master.DefaultEtcdPathPrefix)
+	expEtcdStorage, err2 := master.NewEtcdStorage(etcdClient, "exp/", master.DefaultEtcdPathPrefix)
```

Comment:
Nit: I'd move this down below the err check and use err instead of err2 - more idiomatic